src/sections/Landing/countdown/index.jsx

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect } from "react";
+
+import "./index.scss";
+import { getRemainingTimeUntilMsTimestamp } from "./countdownUtil/index.js";
+
+const defaultRemainingTime = {
+  days: "",
+  hours: "",
+  minutes: "",
+  seconds: "",
+};
+
+// the date in milliseconds
+const date = new Date("08/31/2022 16:00:00"); // some mock date
+const timestampMs = date.getTime();
+
+const CountDown = () => {
+  const [remainingTime, setRemainingTime] = useState(defaultRemainingTime);
+
+  useEffect(() => {
+    const intervalId = setInterval(() => {
+      updateRemainingTime();
+    }, 1000);
+    return () => clearInterval(intervalId);
+  }, []);
+
+  const updateRemainingTime = () => {
+    setRemainingTime(getRemainingTimeUntilMsTimestamp(timestampMs));
+  };
+
+  return (
+    <div className="container">
+      <div>T minus</div>
```

Comment:
What's this T minus part here for?

Reply:
I looked at the wrong design; removed it